src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -124,6 +124,25 @@ def _locate_dir(root_dir, env, kbase_collection, load_ver, check_exists=False, t
     return result_dir
 
 
+def _read_metadata_file(meta_lookup: dict,
+                        genome_id: str) -> dict:
+    # Read the metadata file for a given genome id and return the metadata as a dictionary
+
+    try:
+        meta_filename = meta_lookup[genome_id]
+    except KeyError as e:
+        raise ValueError('Unable to find genome ID') from e
+
+    meta_info = dict()
+    if not pd.isna(meta_filename):
```

Comment:
This happens when the file is empty, which I wouldn't anticipate to happen. I think it was just a safety check.

Reply:
Do you think we will ever run tools directory from NCBI downloaded files? This check is probably necessary if we run tools directory from NCBI files since in that case the metadata file will be empty. But seems like we are planning to run tools only from WS downloaded files, in that case, I can remove this check.

Reply:
👍 removed the check for empty metadatda file.